pkg/webhookadmission/utils.go

```diff
@@ -205,15 +226,23 @@ func createWebhookV1(caBundle []byte, ns string) error {
 		Webhooks: []admissionv1.MutatingWebhook{webhook},
 	}
 
-	// recreate webhook
-	err := admissionregistration.Instance().DeleteMutatingWebhookConfiguration(storkAdmissionController)
-	if err != nil && !k8serr.IsNotFound(err) {
+	// UpdateOrCreate v1 webhook
+	resp, err := admissionregistration.Instance().GetMutatingWebhookConfiguration(storkAdmissionController)
+	if err != nil {
+		if k8serr.IsNotFound(err) {
+			_, err = admissionregistration.Instance().CreateMutatingWebhookConfiguration(req)
+			if err != nil {
+				log.Errorf("Unable to create webhook configuration: %v", err)
+			}
+		}
 		return err
 	}
-	if _, err := admissionregistration.Instance().CreateMutatingWebhookConfiguration(req); err != nil {
-		log.Errorf("unable to create webhook configuration: %v", err)
+	req.ResourceVersion = resp.ResourceVersion
+	if _, err := admissionregistration.Instance().UpdateMutatingWebhookConfiguration(req); err != nil {
+		log.Errorf("Unable to update webhook configuration: %v", err)
 		return err
 	}
+
 	log.Debugf("stork webhook v1 configured: %v", webhookName)
```

Comment:
nit..Although not your change but update it to Stork for consistency if you can.

Reply:
Fixed